src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -207,6 +207,7 @@ class TimeRangeSelector extends React.PureComponent {
       end: null,
       utc: this.state.utc,
     };
+    this.setState(newDateTime);
```

Comment:
Hmm does handleClear need to update state too?

Reply:
Clearing seems to work ok